src/rpm/headers/header.rs

```diff
@@ -311,6 +312,38 @@ where
     }
 }
 
+impl fmt::Display for Header<IndexSignatureTag> {
```

Comment:
Makes it easy to dump a readable summary of the package (the Debug impl isn't very readable due to verbosity)

```
Signature Header - Entries: 11,  Data Section Size: 4212 bytes
===================================================
    HEADER_SIGNATURES: Bin
    RPMSIGTAG_RSA: Bin
    RPMSIGTAG_SHA1: String
    RPMSIGTAG_SHA256: String
    RPMSIGTAG_SIZE: i32
    RPMSIGTAG_PGP: Bin
    RPMSIGTAG_MD5: Bin
    RPMSIGTAG_PAYLOADSIZE: i32
    RPMTAG_INSTALLTIME: Bin
    << UnknownTag >> [5090]: StringArray
    << UnknownTag >> [5091]: i32

Header - Entries: 62,  Data Section Size: 5177 bytes
===================================================
    RPMTAG_HEADERIMMUTABLE: Bin
    RPMTAG_HEADERI18NTABLE: StringArray
    RPMTAG_NAME: String
    RPMTAG_VERSION: String
    RPMTAG_RELEASE: String
    RPMTAG_SUMMARY: I18NString
    RPMTAG_DESCRIPTION: I18NString
    RPMTAG_BUILDTIME: i32
    RPMTAG_BUILDHOST: String
    RPMTAG_SIZE: i32
   ...
   ...
```